core/block_svg.ts

```diff
@@ -671,6 +671,26 @@ export class BlockSvg
     }
   }
 
+  /**
+   * Add a class to the SVG group of this block.
```

Comment:
```suggestion
   * Add a CSS class to the SVG group of this block.
```

For extra clarity :P

Reply:
That makes sense! Updated.